scripts/drone/events/release.star

```diff
@@ -59,6 +60,34 @@ def retrieve_npm_packages_step():
         "commands": ["./bin/build artifacts npm retrieve --tag ${DRONE_TAG}"],
     }
 
+def release_pr_step(depends_on = []):
+    return {
+        "name": "create-release-pr",
+        "image": images["curl"],
+        "depends_on": depends_on,
+        "environment": {
+            "GITHUB_TOKEN": from_secret("github_token"),
+            "GH_CLI_URL": "https://github.com/cli/cli/releases/download/v2.50.0/gh_2.50.0_linux_amd64.tar.gz",
+        },
+        "commands": [
+            "apk add perl",
+            "v_target=`echo $${{TAG}} | perl -pe 's/{}/v\\1.\\2.x/'`".format(semver_regex),
+            "default_target=`if [[ -n $$LATEST ]]; then echo 'main'; else echo $$v_target; fi`",
+            "backport=`if [[ -n $$LATEST ]]; then echo $$v_target; fi`",
+            # Install gh CLI
+            "curl -L $${GH_CLI_URL} | tar -xz --strip-components=1 -C /usr",
+            # Run the release-pr workflow
+            "gh workflow run " +
+            "-f dry_run=$${DRY_RUN} " +
+            "-f version=$${TAG} " +
```

Comment:
Why again are we using `TAG` here instead of `DRONE_TAG` for the version? Looks like a number of other functions are using `DRONE_TAG` for injecting the version. Or maybe I'm reading it wrong.

Reply:
`DRONE_TAG` is set automatically by drone when the pipeline is triggered by a `git tag` event. `TAG` is manually set by folks when promoting a build using the `public` or `private` target.

In the release guide on the "Publish public release" steps, it says,

> Locate the release build for 10.4.3 in on [Drone](https://drone.grafana.net/grafana/grafana-security-mirror).
>     - From the top right menu, select promote.
>     - For target, enter public.
>     - Add a parameter of TAG with value 10.4.3 and click add.
>     - Confirm that the TAG parameter has been added (you will see a remove option for it).
>     - Click Deploy. This will start a new build that will publish the resources for your release.

That's where `TAG` comes from. If you're working on the promotion pipeline, then you have to use `TAG`. if you're working on a tag pipeline, you have to use `DRONE_TAG`.

Reply:
Yep, that makes sense! Thanks.